ui/app/mirrors/edit/[mirrorId]/aggregatedCountsByInterval.ts

```diff
@@ -23,6 +23,15 @@ function aggregateCountsByInterval(
     case 'day':
       timeUnit = 'YYYY-MM-DD';
       break;
+    case '1min':
```

Comment:
Seems like these cases could be merged for things like 15min/5min/1min where result is the same